test/controllers/FileUploadCallbackControllerSpec.scala

```diff
@@ -17,100 +17,88 @@
 package controllers
 
 import akka.stream.Materializer
-import models.{CallbackData, ErsMetaData, SchemeInfo}
-import org.joda.time.DateTime
-import org.mockito.ArgumentMatchers
+import models.upscan._
+import org.mockito.ArgumentCaptor
+import org.mockito.ArgumentMatchers.{eq => meq, _}
 import org.mockito.Mockito._
 import org.scalatest.mockito.MockitoSugar
 import org.scalatestplus.play.{OneAppPerSuite, PlaySpec}
+import play.api.Application
 import play.api.inject.guice.GuiceApplicationBuilder
-import play.api.libs.json.{JsObject, JsValue, Json}
+import play.api.libs.json._
 import play.api.mvc.Request
+import play.api.test.FakeRequest
 import play.api.test.Helpers._
-import play.api.test.{FakeHeaders, FakeRequest}
-import play.api.{Application, Configuration}
 import services.SessionService
-import uk.gov.hmrc.http.cache.client.CacheMap
-import uk.gov.hmrc.play.frontend.auth.connectors.AuthConnector
-import utils.{CacheUtil, ERSFakeApplicationConfig, Fixtures}
+import uk.gov.hmrc.http.HeaderCarrier
+import utils.{ERSFakeApplicationConfig, UpscanData}
 
 import scala.concurrent.Future
-import uk.gov.hmrc.http.HeaderCarrier
 
-class FileUploadCallbackControllerSpec extends PlaySpec with MockitoSugar with ERSFakeApplicationConfig with OneAppPerSuite {
+class FileUploadCallbackControllerSpec extends PlaySpec with MockitoSugar with ERSFakeApplicationConfig with OneAppPerSuite with UpscanData {
 
   override implicit lazy val app: Application = new GuiceApplicationBuilder().configure(config).build()
   implicit lazy val mat: Materializer = app.materializer
 
-  lazy val mockAuthConnector = mock[AuthConnector]
-  lazy val mockCurrentConfig = mock[Configuration]
-  lazy val mockSessionService = mock[SessionService]
-  lazy val mockCacheUtil = mock[CacheUtil]
+  lazy val mockSessionService: SessionService = mock[SessionService]
```

Comment:
does this need to be lazy, as its not reliant on app

Reply:
It doesnt, Ive changed this to be a val